tests/e2e/v3_curl_test.go

```diff
@@ -288,7 +288,7 @@ func testV3CurlCampaign(cx ctlCtx) {
 		Endpoint: path.Join(cx.apiPrefix, "/election/campaign"),
 		Value:    string(cdata),
 	})
-	lines, err := e2e.SpawnWithExpectLines(cargs, cx.envMap, `"leader":{"name":"`)
+	lines, err := e2e.SpawnWithExpectLines(context.TODO(), cargs, cx.envMap, `"leader":{"name":"`)
```

Comment:
Minor comment: have you considered creating a context with timeout?

Reply:
> I think this PR needs to be merge asap, otherwise you have to resolve conflict frequently.

Due to the same resonances I didn't want to fix everything at once. Also I don't care much about old style/`ctlCtx` tests as they will be rewritten when migrated to common framework.